package.json

```
@@ -10,20 +10,29 @@
     "node": "^16.6.1",
     "npm": ">=7.20.0"
   },
-  "main": "index.js",
+  "exports": {
+    "./style/standard": "./style/standard.js",
+    "./style/verbose": "./style/verbose.js",
+    "./correctness": "./correctness/index.js"
```

does it work if you do:
```suggestion
    "./correctness": "./correctness"
```